src/stan_math_backend/Transform_Mir.ml

```diff
@@ -46,10 +46,18 @@ let opencl_supported_functions =
   ; "poisson_lpmf"; "poisson_log_lpmf"; "poisson_log_glm_lpmf"; "rayleigh_lpdf"
   ; "scaled_inv_chi_square_lpdf"; "skew_normal_lpdf"; "std_normal_lpdf"
   ; "student_t_lpdf"; "uniform_lpdf"; "weibull_lpdf" ]
+  @ [ "beta"; "crossprod"; "cholesky_decompose"; "digamma"; "inv_cloglog"
+    ; "inv_Phi"; "lbeta"; "lgamma"; "log1m_exp"; "log1m_inv_logit"; "log1p_exp"
+    ; "log_diff_exp"; "log_inv_logit"; "log_inv_logit_diff"
+    ; "mdivide_left_tri_low"; "mdivide_right_tri_low"; "multiply"
+    ; "rows_dot_product"; "rows_dot_self"; "tcrossprod"; "tgamma" ]
```

Comment:
The list of supported functions is longer (see Supported functions
 in http://mc-stan.org/math/opencl_support.html) but these are typically faster standalone (with data transfers) as shown by Math benchmarks.